lib/editor/util/__tests__/__snapshots__/map.js.snap

```diff
@@ -0,0 +1,1244 @@
+      "timepoint": null,
+    },
+    Object {
+      "distance": 345.8645390523855,
+      "id": "y1e5",
+      "permanent": true,
+      "point": Object {
+        "geometry": Object {
+          "coordinates": Array [
+            -77.92215494185281,
+            42.08925407008939,
+          ],
+          "type": "Point",
+        },
+        "properties": Object {},
+        "type": "Feature",
+      },
+    },
+    Object {
+      "defaultDwellTime": 0,
+      "defaultTravelTime": 120,
+      "distance": 691.729078104771,
+      "hidden": true,
+      "id": "fyz2",
+      "permanent": true,
+      "point": Object {
+        "geometry": Object {
+          "coordinates": Array [
+            -77.92252574804179,
+            42.09235127309673,
+          ],
+          "type": "Point",
+        },
+        "properties": Object {},
+        "type": "Feature",
+      },
+      "shapeDistTraveled": 691.729078104771,
+      "stopId": "a9c9cd55-ca3f-4ce7-9a93-96f2f4eaf089",
+      "timepoint": null,
+    },
+  ],
+}
+`;
+
+exports[`editor > util > map > recalculateShape > deleting > should delete the last stop when there are control points after the last stop 1`] = `
+Object {
+  "coordinates": Array [
+    Array [
+      Array [
+        -77.922203,
+        42.086407,
+      ],
+      Array [
+        -77.922196,
+        42.086498,
+      ],
+      Array [
+        -77.922173,
+        42.086498,
+      ],
+      Array [
+        -77.92183700000001,
+        42.086513000000004,
+      ],
+    ],
+    Array [
+      Array [
+        -77.92183700000001,
+        42.086513000000004,
+      ],
+      Array [
+        -77.92188300000001,
+        42.086956,
+      ],
+      Array [
+        -77.921929,
+        42.087368,
+      ],
+      Array [
+        -77.922043,
+        42.088386,
+      ],
+      Array [
+        -77.922112,
+        42.088844,
+      ],
+      Array [
+        -77.922135,
+        42.089099,
+      ],
+    ],
+    Array [
+      Array [
+        -77.922135,
+        42.089099,
+      ],
+      Array [
+        -77.92231799999999,
+        42.090522,
+      ],
+      Array [
+        -77.92235600000001,
+        42.090847000000004,
+      ],
+      Array [
+        -77.922432,
+        42.091491000000005,
+      ],
+      Array [
+        -77.922509,
+        42.092201,
+      ],
+      Array [
+        -77.922654,
+        42.093502,
+      ],
+      Array [
+        -77.922684,
+        42.093745999999996,
+      ],
+    ],
+    Array [
+      Array [
+        -77.922684,
+        42.093745999999996,
+      ],
+      Array [
+        -77.922707,
+        42.09391,
+      ],
+      Array [
+        -77.92274499999999,
+        42.094158,
+      ],
+      Array [
+        -77.922799,
+        42.094528000000004,
+      ],
+      Array [
+        -77.922974,
+        42.095657,
+      ],
+      Array [
+        -77.923104,
+        42.096481,
+      ],
+    ],
+  ],
+  "updatedControlPoints": Array [
```

Comment:
This snapshot makes no sense due to bad input data. The pattern coordinates are in NY while the control point coordinates are in CA.